lib/wca_live/synchronization/export.ex

```diff
@@ -208,4 +214,37 @@ defmodule WcaLive.Synchronization.Export do
       # "scrambleSetId" => nil # ignored for now
     }
   end
+
+  defp competition_to_results(competition) do
+    competition.competition_events |> Enum.map(&competition_event_for_results/1)
+  end
+
+  defp competition_event_for_results(competition_event) do
+    %{
+      "id" => competition_event.event_id,
+      "rounds" => competition_event.rounds |> Enum.map(&round_for_results/1)
+    }
+  end
+
+  defp round_for_results(round) do
+    %{
+      "id" => round_wcif_id(round),
+      "roundNumber" => round.number,
+      "cutoff" => round.cutoff && round.cutoff |> cutoff_to_wcif(),
+      "timeLimit" => round.time_limit && round.time_limit |> time_limit_to_wcif(),
```

Comment:
I wouldn't include cutoff/timelimit. I'm thinking the response could be this:

```json
{
  "rounds": [
    {
      "eventId": "333",
      "number": 1,
      "results": [
        {
          "personId": 1,
          "best": 100,
          "average": 100,
          "attemptResults": [100, 100, 100, 100, 100]
        }
      ]
    }
  ],
  "persons": [
    {
      "id": 1,
      "wcaId": "2000FOOO01",
      "name": "Foo Bar"
    }
  ]
}
```

It's worth extracting the person data, so there is less duplication.